charts/grafana/values.yaml

```diff
@@ -1534,17 +1534,31 @@ networkPolicy:
 # Enable backward compatibility of kubernetes where version below 1.13 doesn't have the enableServiceLinks option
 enableKubeBackwardCompatibility: false
 useStatefulSet: false
-# Create a dynamic manifests via values:
+
```

Comment:
The comment should come after `extraObjects: []`

Reply:
No problem. Should the one line description be above `extraObjects: []` and the examples below or everything below?